### DISCUSSION TITLE
Integrating custom L1 layer with srsRAN CU/DU

### DISCUSSION THREAD

**Discussion by abcd11abcd** [srsran/srsRAN_Project#660] [General Help]
I want to use srsRAN project without phy layer. In the place of srsRAN's phy layer, I want to use a custom L1 layer (this L1 also uses similar shared memory approach as FAPI between MAC and PHY). In this regard, I want to know how to compile srsRAN without phy layer. Kindly help me how to proceed with this.
If I'm able to compile this project till MAC layer, then I will try to integrate my L1 to this project and test. Any leads on doing this are welcome.

I want to know whether srsRAN support 3rd party PHY layers and FAPI interfaces? Please help me.

**Comment by faluco** [comment]
Hi @abcd11abcd,

We have already integrated srsRAN with other 3rd party PHY vendors using split 6 (FAPI), so it's absolutely possible to do it. This specific feature is fully supported in the srsRAN Enterprise project, so please contact sales@srs.io for further assistance.

**Comment by abcd11abcd** [comment]
okay. Thanks for the information.
